package.json

```diff
@@ -129,7 +129,6 @@
     "jest": "^23.6.0",
     "jest-axe": "^3.1.0",
     "json-stable-stringify-without-jsonify": "^1.0.1",
-    "leven": "^2.1.0",
```

Comment:
`leven` package was used in SUIR for [IconSearch](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/docs/src/components/IconSearch/IconSearch.js) component, we don't have such.

_Why? In production mode, i.e. hosted docs `leven()` function from `lib` was nooped_